src/lib.rs

```diff
@@ -411,6 +411,64 @@ impl<'text> BidiInfo<'text> {
         result.into()
     }
 
+    /// Reorder the levels
+    ///   # # Example
+    /// ```
+    /// use unicode_bidi::BidiInfo;
+    /// use unicode_bidi::Level;
+    ///
+    /// let l0 = Level::from(0);
+    /// let l1 = Level::from(1);
+    /// let l2 = Level::from(2);
+    /// let levels: Vec<Level> = vec![l0, l0, l0, l1, l1, l1, l2, l2];
```

Comment:
nit: I think this example should probably use `.visual_runs()` or something to demonstrate proper usage. The user should probably not be constructing their own type here.